code/game/objects/items/devices/radio/intercom.dm

```diff
@@ -174,6 +175,43 @@ pixel_x = 8;
 	set_broadcasting(TRUE)
 	set_listening(FALSE)
 
+/obj/item/device/radio/intercom/expedition
+	name = "intercom (expeditionary)"
+
+/obj/item/device/radio/intercom/expedition/north
+	PRESET_NORTH
+
+/obj/item/device/radio/intercom/expedition/south
+	PRESET_SOUTH
+
+/obj/item/device/radio/intercom/expedition/west
+	PRESET_WEST
+
+/obj/item/device/radio/intercom/expedition/east
+	PRESET_EAST
+
+/obj/item/device/radio/intercom/expedition/Initialize()
+	. = ..()
+	set_frequency(EXP_FREQ)
+	internal_channels = default_expedition_channels
+
+/obj/item/device/radio/intercom/expedition/hailing/north
+	PRESET_NORTH
+
+/obj/item/device/radio/intercom/expedition/hailing/south
+	PRESET_SOUTH
+
+/obj/item/device/radio/intercom/expedition/hailing/west
+	PRESET_WEST
+
+/obj/item/device/radio/intercom/expedition/hailing/east
+	PRESET_EAST
+
+/obj/item/device/radio/intercom/expedition/hailing/Initialize()
```

Comment:
```suggestion
/obj/item/device/radio/intercom/hailing/Initialize()
```
Is there anything that makes this radio specific to expeditions?
-> At a quick glance it looks like a normal radio with the hailing channel enabled.

Reply:
Expedition intercoms can only be used to tune into expedition and hailing channels, so a physician can't tune it into medical comms or any other non-public channel. The difference between the regular and hailing versions here is that the hailing one starts tuned to hailing so the Intrepid and Spark can hear any hails made to them.